sdk/communication/communication-job-router/test/public/methods/classificationPolicies.spec.ts

```diff
@@ -0,0 +1,110 @@
+
+      await client.createDistributionPolicy(distributionPolicyRequest.id!, {
+        patch: distributionPolicyRequest
+      });
+      await client.createExceptionPolicy(exceptionPolicyRequest.id!, {
+        patch: exceptionPolicyRequest
+      });
+      await client.createQueue(queueRequest.id!, { patch: queueRequest });
+    });
+
+    afterEach(async function(this: Context) {
+      if (!this.currentTest?.isPending() && recorder) {
+        // unused
+      }
+
+      await client.deleteClassificationPolicy(classificationPolicyRequest.id!, {});
+    });
+
+    this.afterAll(async function(this: Context) {
+      await client.deleteClassificationPolicy(classificationPolicyRequest.id!, {});
+      await client.deleteQueue(queueRequest.id!, {});
+      await client.deleteExceptionPolicy(exceptionPolicyRequest.id!, {});
+      await client.deleteDistributionPolicy(distributionPolicyRequest.id!, {});
+    });
+
+    it("should create a classification policy", async function() {
+      const result = await client.createClassificationPolicy(request.id!, { patch: request });
+
+      assert.isDefined(result);
+      assert.isDefined(result.id);
+      assert.equal(result.name, request.name);
+    }).timeout(timeoutMs);
+
+    it("should get a classification policy", async function() {
+      const response: ClassificationPolicy = await client.createClassificationPolicy(request.id!, {
+        patch: request
+      });
+
+      const result = await client.getClassificationPolicy(request.id!);
+
+      assert.equal(result.id, response.id);
+      assert.equal(result.name, response.name);
+      assert.equal(result.fallbackQueueId, response.fallbackQueueId);
+      assert.deepEqual(result.queueSelectors, response.queueSelectors);
+      assert.deepEqual(result.prioritizationRule, response.prioritizationRule);
+      assert.deepEqual(result.workerSelectors, response.workerSelectors);
+
+      // Should we just do this instead?
+      //     assert.deepEqual(result, response);
+    }).timeout(timeoutMs);
+
+    it("should update a classification policy", async function() {
+      var response: ClassificationPolicy = await client.createClassificationPolicy(request.id!, {
+        patch: request
+      });
+
+      const patch = { ...response, name: "new name" };
+      const result = await client.updateClassificationPolicy(response.id!, { patch });
+
+      assert.isDefined(result);
+      assert.isDefined(result.id);
+      assert.equal(result.name, patch.name);
+    }).timeout(timeoutMs);
+
+    it("should delete a classification policy", async function() {
+      const response: ClassificationPolicy = await client.createClassificationPolicy(request.id!, {
+        patch: request
+      });
+
+      const result = await client.deleteClassificationPolicy(response.id!, {});
+
+      assert.isDefined(result); // TODO: What validation criteria are there for a delete call?
```

Comment:
@marche0133 question here.

Reply:
For deletion, if you really want to check, you can call get request with id again, and it should throw Not Found error. otherwise as long as response is StatusOk, then it is fine.